packages/toolkit/src/shared/cloudformation/cloudformation.ts

```diff
@@ -19,7 +19,8 @@ export const SERVERLESS_API_TYPE = 'AWS::Serverless::Api' // eslint-disable-line
 export const SERVERLESS_FUNCTION_TYPE = 'AWS::Serverless::Function' // eslint-disable-line @typescript-eslint/naming-convention
 export const LAMBDA_FUNCTION_TYPE = 'AWS::Lambda::Function' // eslint-disable-line @typescript-eslint/naming-convention
 
-export const templateFileGlobPattern = '**/*.{yaml,yml}'
+export const templateFileGlobPattern = '**/*.{yaml,yml,json,template}'
+export const templateFileRegexPattern = /.*\.(yaml|yml|json|template)$/i
```

Comment:
Thanks for analyzing relevant codepaths and making informed decisions about them!